src/libNode/MicroBlockPreProcessing.cpp

```diff
@@ -792,6 +809,10 @@ void Node::ReinstateMemPool(
     m_unconfirmedTxns.emplace(t.GetTranID(),
                               PoolTxnStatus::PRESENT_GAS_EXCEEDED);
   }
+
+  for (const auto& txnHashStatus : droppedTxns) {
```

Comment:
This could be a good opportunity for move semantics ([vector example](https://stackoverflow.com/questions/15004517/moving-elements-from-stdvector-to-another-one))

Reply:
I tried doing this and turns out you cannot add vector into a map this way. Moreover `emplace` already uses move semantics